mono/mini/intrinsics.c

```diff
@@ -1161,6 +1161,26 @@ mini_emit_inst_for_method (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSign
 		 * all inputs:
 		 * http://everything2.com/?node_id=1051618
 		 */
+#ifdef MONO_ARCH_SIMD_INTRINSICS
+		if (!cfg->compile_aot && (mono_arch_cpu_enumerate_simd_versions () & SIMD_VERSION_SSE41) && fsig->param_count == 1 && fsig->params [0]->type == MONO_TYPE_R8) {
```

Comment:
This looks wrong, it should be in the amd64 backend.

Reply:
All other simd code is in simd-intrinsics.c as well.